resources/views/formfields/multiple_checkbox.blade.php

```diff
@@ -13,7 +13,9 @@
         @endif
 
         <input type="checkbox" name="{{ $row->field }}[{{$key}}]"
-               {!! $checked ? 'checked="checked"' : '' !!} value="{{$key}}" />
-        {{$label}}
+        {!! $checked ? 'checked="checked"' : '' !!} value="{{$key}}" 
```

Comment:
Can you return the formatting back to inline attributes?  No need to change that